docs/wiki/Client-applications.md

```diff
@@ -0,0 +1,30 @@
+## Client Applications and the Core Package
```

Comment:
So, clearly wiki needs work.  I'm not sure where this section I extracted from README belongs on the wiki since it's a mix of user application overview and an intro to the `core` package.  I'm just taking it out of readme and putting it somewhere in wiki for more extensive revision.